src/main/kotlin/com/example/fhirvalidator/shared/HybridTerminologyValidationSupport.java

```diff
@@ -0,0 +1,193 @@
+package com.example.fhirvalidator.shared;
```

Comment:
i strongly dont feel like we should copy files from a library and check them into our code base. is there a reason we are not making use of the library

Reply:
Commented elsewhere 

Don't intend to keep this code in our project, long term move to HAPI FHIR